typescript/src/events/messages/guildMessageDeleteNotify.ts

```diff
@@ -3,39 +3,38 @@ import { LanguageKeys } from '#lib/i18n/languageKeys';
 import type { GuildMessage } from '#lib/types';
 import { Colors } from '#lib/types/Constants';
 import { Events } from '#lib/types/Enums';
-import { MessageLogsEnum } from '#utils/constants';
 import { getContent, getImage } from '#utils/util';
 import { ApplyOptions } from '@sapphire/decorators';
 import { Event, EventOptions } from '@sapphire/framework';
 import { cutText } from '@sapphire/utilities';
-import { MessageEmbed, TextChannel } from 'discord.js';
+import { MessageEmbed } from 'discord.js';
 
 @ApplyOptions<EventOptions>({ event: Events.GuildMessageDelete })
 export class UserEvent extends Event {
 	public async run(message: GuildMessage) {
-		const [enabled, ignoredChannels, ignoredDeletes, ignoredAll, t] = await message.guild.readSettings((settings) => [
-			settings[GuildSettings.Events.MessageDelete],
+		const key = GuildSettings.Channels.Logs[message.channel.nsfw ? 'MessageDeleteNsfw' : 'MessageDelete'];
+		const [ignoredChannels, channelId, ignoredDeletes, ignoredAll, t] = await message.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [ignoredChannels, logChannelId, ignoredDeletes, ignoredAll, t] = await message.guild.readSettings((settings) => [
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.